--- contrib/gis/db/backends/postgis/creation.py
+++ contrib/gis/db/backends/postgis/creation.py
@@ -38,12 +38,17 @@ class PostGISCreation(DatabaseCreation):
                                   style.SQL_FIELD(qn(f.column)) +
                                   style.SQL_KEYWORD(' SET NOT NULL') + ';')
 
-
             if f.spatial_index:
                 # Spatial indexes created the same way for both Geometry and
                 # Geography columns
+                # PostGIS 2.0 does not support GIST_GEOMETRY_OPS
                 if f.geography:
                     index_opts = ''
+                elif self.connection.ops.spatial_version >= (2, 0):
+                    if f.dim > 2:
+                        index_opts = ' ' + style.SQL_KEYWORD('gist_geometry_ops_nd')
+                    else:
+                        index_opts = ''
                 else:
                     index_opts = ' ' + style.SQL_KEYWORD(self.geom_index_opts)
                 output.append(style.SQL_KEYWORD('CREATE INDEX ') +
